typeshed/__init__.py

```diff
@@ -1,4 +1,53 @@
-import os
+# -*- coding: utf-8 -*-
+"""Typeshed is a collection of type annotations for the standard library and
+third-party libraries.
+
+The type annotations in question are stored in so-called "stub files".  They're
+Python-like files with the *.pyi file extension.  Those files are parseable by
+the built-in Python AST but are not executable.  Functions and methods in stub
+files don't contain bodies.  Variables and attributes don't contain real values.
+
+Typeshed is organized in the following directory structure:
+
+    typeshed
+    ├── __init__.py
+    ├── stdlib
+    │   ├── 2
+    │   ├── 2and3
+    │   ├── 3
+    │   ├── 3.3
+    │   ├── 3.4
+    │   └── ...
+    └── third_party
+        ├── 2
+        ├── 2and3
+        └── 3
```

Comment:
There can be 3.4 etc. here too (though I'm elated there aren't any currently :-).